src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs

```diff
@@ -1506,7 +1506,7 @@ LIMIT 1
 SELECT pch.*
 FROM most_recent_package_codes mrpc
 INNER JOIN package_code_history pch
-ON pch.id = ANY(mrpc.package_code_ids)
+ON pch.id = ANY(mrpc.package_code_ids) AND pch.is_deleted = false
```

Comment:
This `AND is_deleted = false` is nothing more but a sanity check, isn't it? Deleted entries should not be present in `package_code_ids` anyway, aren't they? Do we use such sanity check elsewhere?

Reply:
yes it's just santiy check. we have sthn similar in metadata but not for others. I'm fine by removing that if you want.